workspace.json

```diff
@@ -0,0 +1,170 @@
+    "path": "packages/libs/chainweb-node-client"
+  },
+  {
+    "name": "@kadena/chainweb-stream-client",
+    "version": "0.0.8",
+    "private": false,
+    "path": "packages/libs/chainweb-stream-client"
+  },
+  {
+    "name": "@kadena/chainwebjs",
+    "version": "0.0.5",
+    "private": false,
+    "path": "packages/libs/chainwebjs"
+  },
+  {
+    "name": "@kadena/client",
+    "version": "1.0.0-alpha.3",
+    "private": false,
+    "path": "packages/libs/client"
+  },
+  {
+    "name": "@kadena/client-examples",
+    "version": "0.0.1",
+    "private": true,
+    "path": "packages/libs/client-examples"
+  },
+  {
+    "name": "@kadena/cryptography-utils",
+    "version": "0.3.6",
+    "private": false,
+    "path": "packages/libs/cryptography-utils"
+  },
+  {
+    "name": "kadena.js",
+    "version": "0.0.1",
+    "private": false,
+    "path": "packages/libs/kadena.js"
+  },
+  {
+    "name": "@kadena/pactjs",
+    "version": "0.3.0",
+    "private": false,
+    "path": "packages/libs/pactjs"
+  },
+  {
+    "name": "@kadena/pactjs-generator",
+    "version": "1.0.0-alpha.3",
+    "private": false,
+    "path": "packages/libs/pactjs-generator"
+  },
+  {
+    "name": "@kadena/react-components",
+    "version": "0.0.1",
+    "private": false,
+    "path": "packages/libs/react-components"
+  },
+  {
+    "name": "@kadena/react-ui",
+    "version": "0.0.1",
+    "private": false,
+    "path": "packages/libs/react-ui"
+  },
+  {
+    "name": "@kadena/types",
+    "version": "0.4.0",
+    "private": false,
+    "path": "packages/libs/types"
+  },
+  {
+    "name": "@kadena/cookbook",
+    "version": "1.0.0",
+    "private": true,
+    "path": "packages/tools/cookbook"
+  },
+  {
+    "name": "@kadena/create-kadena-app",
+    "version": "0.1.0",
+    "private": false,
+    "path": "packages/tools/create-kadena-app"
+  },
+  {
+    "name": "@kadena-dev/eslint-config",
+    "version": "0.0.10",
+    "private": false,
+    "path": "packages/tools/eslint-config"
+  },
+  {
+    "name": "@kadena-dev/eslint-plugin",
+    "version": "0.0.5",
+    "private": false,
+    "path": "packages/tools/eslint-plugin"
+  },
+  {
+    "name": "@kadena-dev/heft-rig",
+    "version": "0.0.10",
+    "private": false,
+    "path": "packages/tools/heft-rig"
+  },
+  {
+    "name": "@kadena/integration-tests",
+    "version": "0.0.1",
+    "private": true,
+    "path": "packages/tools/integration-tests"
+  },
+  {
+    "name": "@kadena/kda-cli",
+    "version": "0.0.1",
+    "private": false,
+    "path": "packages/tools/kda-cli"
+  },
+  {
+    "name": "@kadena/pactjs-cli",
+    "version": "1.0.0-alpha.3",
+    "private": false,
+    "path": "packages/tools/pactjs-cli"
+  },
+  {
+    "name": "@kadena-dev/markdown",
+    "version": "1.0.0",
+    "private": true,
+    "path": "packages/tools/remark-plugins"
+  },
+  {
+    "name": "@kadena-dev/rush-fix-versions",
+    "version": "0.0.6",
+    "private": false,
+    "path": "packages/tools/rush-fix-versions"
+  }
+]
```

Comment:
What is this file for?

Reply:
It's a generated replacement for `rush.json#projects`, can be used to generate other things. Currently it's only used to generate the tabel in the root README.md.

Generated using some crazy script (chatgpt ftw!) in `pnpm _list`. We should make things more clear here, wasn't sure yet how.